File: arkts_static_test/03_Types/ets_cases/3.17.1_Readonly_Tuple_Types/cross_lang_verify/runtime/TYP_03_17_01_008.swift
/**
 * Swift cross-language verification for TYP_03_17_01_008_RUNTIME_READONLY_TUPLE_LENGTH
 */
import Foundation

let tuple: (Int, String, Bool) = (1, "hello", true)
let mirror = Mirror(reflecting: tuple)

if mirror.children.count != 3 { fatalError("length should be 3") }

print("verified")

File: arkts_static_test/03_Types/ets_cases/3.17.1_Readonly_Tuple_Types/cross_lang_verify/runtime/TYP_03_17_01_008.java
/**
 * Java cross-language verification for TYP_03_17_01_008_RUNTIME_READONLY_TUPLE_LENGTH
 */
class TYP_03_17_01_008 {
    public static void main(String[] args) {
        final Object[] tuple = {1, "hello", true};

        if (tuple.length != 3) throw new AssertionError("length should be 3");

        System.out.println("verified");
    }
}
